tests/app/dao/test_jobs_dao.py

```diff
@@ -351,17 +351,21 @@ def test_should_get_jobs_seven_days_old_by_scheduled_for_date(sample_service):
 
 @freeze_time("2016-10-31 10:00:00")
 def test_should_get_limited_number_of_jobs(sample_template):
-    flexable_retention_service = create_service(service_name="Another service")
-    insert_service_data_retention(flexable_retention_service.id, sample_template.template_type, 3)
-    flexable_template = create_template(flexable_retention_service, template_type=sample_template.template_type)
+    flexable_retention_service1 = create_service(service_name="Another service 1")
```

Comment:
What is `flexable`? or did we mean `flexible`?

Reply:
🤔 In my defence I introduced this spelling mistake in a previous PR so I blame Past Steve.